src/server/game/Spells/SpellEffects.cpp

```diff
@@ -3822,6 +3822,12 @@ void Spell::EffectScriptEffect(SpellEffIndex effIndex)
                             m_caster->CastSpell(unitTarget, 22682, true);
                             return;
                         }
+                    // Jubling Cooldown
+                    case 23853:
+                        {
+                            unitTarget->CastSpell(unitTarget, 23852, false);
+                        }
+                        break;
```

Comment:
Properly script the spell or check if spell_linked_spell table works for this one.